perception:
  ros__parameters:
    #Topic
    lidar_topic: "/fast_limo/pointcloud"
    state_topic: "/car_state/state"

    #Parameters of FOV
    crop: true
    max_x_fov: 1500.0
    max_y_fov: 1500.0
    max_z_fov: 5.0

    #Parameters of ground filter
    threshold_ground_filter: 0.075

    number_sections: 8
    angle_threshold: 35.0
    minimum_ransac_points: 30

    #Parameters of clustering
    radius: 0.15

    #Parameters of scoring
    threshold_scoring: 0.10

    #Parameters of coloring
    distance_threshold: 5.0
    coloring_threshold: 15.0